app/models/user.rb

```diff
@@ -318,9 +318,21 @@ def email_as_id
     self.email.gsub(/[@\.]/, '-')
   end
 
-  # return branding groups visible to user (or all for admins)
+  # return branding groups available to user to add studies to (or all for admins)
   def available_branding_groups
-    self.admin? ? BrandingGroup.all.order_by(:name.asc) : BrandingGroup.where(user_id: self.id).order_by(:name.asc)
+    if self.admin?
+      BrandingGroup.all.order_by(:name.asc)
+    else
+      BrandingGroup.where(user_id: self.id).order_by(:name.asc)
+    end
+  end
+
+  def visible_branding_groups
+    if self.admin?
+      BrandingGroup.all.order_by(:name.asc)
+    else
+      BrandingGroup.where(user_id: self.id).or(public: true).order_by(:name.asc)
```

Comment:
Blocker: chaining these queries together will not have the intended outcome.  This is going to evaluate to "all branding groups owned by the user that ARE ALSO public".  If what you're looking to do is "give me all branding groups that are public OR owned by the user", then it will need to be written as:
```
BrandingGroup.or(public: true, user_id: self.id).order_by(:name.asc)
```
You can also use `any_of` as it is an alias for `or`, which I personally find a little more readable.  To make this a little DRYer, I'd write it as:

```
selector = self.admin? ? BrandingGroup.all : BrandingGroup.any_of(public: true, user_id: self.id)
selector.order_by(name: :asc)
```

Reply:
I'm pretty sure this is how `or` works. See `https://www.bigbinary.com/blog/rails-5-adds-or-support-in-active-record`   Locally, you can try something like this to confirm the behavior:
```2.6.6 :022 > Study.where(accession:'SCP41').or(accession: 'SCP29').count
 => 2 
```

Reply:
Huh.  Learn something every day.  I'll approve based off of that since that was my only request.